docs/en/reference/events.rst

```diff
@@ -240,6 +240,8 @@ a relevant lifecycle event. More than one callback can be defined for each
 lifecycle event. Lifecycle Callbacks are best used for simple operations
 specific to a particular entity class's lifecycle.
 
+Note that Licecycle Callbacks can not be defined for Embeddables.
+
```

Comment:
If you use this syntax it will render it as a note box:

    .. note::

        Note that Licecycle Callbacks can not be defined for Embeddables.

Reply:
Yeah, I saw it. I was not sure if it is *that* important to mention this. Anyhow, added it!